src/PIL/PngImagePlugin.py

```diff
@@ -1110,7 +1110,10 @@ def _write_multiple_frames(im, fp, chunk, rawmode):
 
     # animation control
     chunk(
-        fp, b"acTL", o32(len(im_frames)), o32(loop),  # 0: num_frames  # 4: num_plays
+        fp,
+        b"acTL",
+        o32(len(im_frames)),
+        o32(loop),  # 0: num_frames  # 4: num_plays
```

Comment:
Yup, before [this](https://github.com/python-pillow/Pillow/commit/512b060a53cc189770646414ce91e552eb01fe2b#diff-07cbf160cd247acb9b67060ff3d81ed9L1074), used to be like this:
 
```suggestion
        fp,
        b"acTL",
        o32(len(im_frames)),  # 0: num_frames
        o32(loop),  # 4: num_plays
```